game/game/playercontrol.cpp

```diff
@@ -1031,6 +1031,16 @@ void PlayerControl::quitPicklock(Npc& pl) {
   }
 
 void PlayerControl::assignRunAngle(Npc& pl, float rotation, uint64_t dt) {
+  //Only assign run angle if player is running (to prevent weird angles when stationary)
+  if(pl.isInState(ScriptFn()) && pl.isAttackAnim() && !pl.isFinishingMove()) {
+    runAngleDest = 0;
+    return;
+  }
+  if (pl.animMoveSpeed(0).x < 0 || pl.animMoveSpeed(0).z < 0) {
```

Comment:
There is already and explicit check for `ani==Npc::Anim::Move`, in place where function is called.
Also, calling `animMoveSpeed` with delta-time equals zero - this condition has no chance to be `true`.